android/app/src/main/java/hr/foi/air/wattsup/viewmodels/ChargerViewModel.kt

```diff
@@ -26,16 +33,21 @@ class ChargerViewModel : ViewModel() {
     private val _currentChargeAmount: MutableLiveData<Float> =
         MutableLiveData(_initialChargeAmount.value)
     private val _percentageChargedUntilFull = MutableLiveData(0f)
+    private val _currentChargeVolume = MutableLiveData(0f)
 
     val charging: LiveData<Boolean> get() = _charging
     val timeElapsed: LiveData<Long> get() = _timeElapsed
 
     val percentageChargedUntilFull: LiveData<Float> get() = _percentageChargedUntilFull
     val amountNecessaryForFullCharge: LiveData<Float> get() = _amountNecessaryForFullCharge
     val currentChargeAmount: LiveData<Float> get() = _currentChargeAmount
+    val currentChargeVolume: LiveData<Float> get() = _currentChargeVolume
```

Comment:
This variable serves the same purpose as `_currentChargeAmount` (which is clamped between 0 and 1 as it is meant to show percentage of full charge at any given moment, but can be just multiplied by some number to simulate kWh as well before being sent to backend). Also, setting this variable to 0f in `startCharging` function will always send 0f to backend regardless of the car's current charge amount. (It shouldn't be initialised with 0f here either, but rather with `_initialChargeAmount.value` like `currentChargeAmount`, since the car doesn't have to be empty but can e.g. initially be charged halfway when it starts charging for the first time).

The simplest is to remove all references of `currentChargeVolume` and to use `_currentChargeAmount` for EventBody, e.g. like this

```
val chargeVolume = _currentChargeAmount.value!! * maxBatteryCapacityKWH
val eventBody = EventBody(1, UserCard.userCard.value!!.id, chargeVolume)
```

and to put `maxBatteryCapacityKWH `at the top of the ViewModel as e.g. 40, assuming that is taken as average EV battery capacity